Yeti-experience/draft-song-yeti-testbed-experience-02.xml

```diff
@@ -522,87 +540,63 @@ yeti-ns.switch.ch              3600000    IN   AAAA     2001:620:0:ff::29
-   servers.net" addresses in the additional section, because root
-   servers not only answer for root zone, but also answer for "root-
-   servers.net" zone.  Note that BIND will not behave like this if it is
-   not configured for the "root-servers.net" zone.  NSD and Knot happily
-   return such "glue" in the additional section, whether configured for
-   the "root-servers.net" zone or not.</t>
-    
-      <t>The Yeti root naming scheme uses separate and independent domain for
-   individual root servers.  It this case, the priming response from
-   Yeti root servers will only contain the A &amp; AAAA records of that
-   responding server in BIND 9.  However it is desired that the Yeti root servers
-   respond to priming queries with the addresses of all Yeti root
-   servers in the additional section.  This will make them operate as
-   similar to the IANA root servers as possible.</t>
-
-      <t>In Yeti root system, there are two approaches adopted in different
-   root servers.  One is to patch BIND 9 so that it includes the glue
-   addresses in the additional section.  The other one is to add a zone
-   file for each root server and answer for all of them at each Yeti
-   server.  That means each Yeti root server would have a small zone
-   file for "bii.dns-lab.net", "yeti-ns.wide.ad.jp", "yeti-ns.tisf.net",
-   and so on.</t>
+   was not fixed.  Current IANA Root server adopt [a-m].root-servers.net naming 
+   scheme to represent 13 servers which are labeled with letter from A to M.
+   One reason behind this naming scheme is that DNS label compression can be used 
+   to produce a smaller DNS response within 512 bytes.  But in Yeti testbed there is 
+   a chance to design and test alternative naming schemes carrying following desired 
+   properties.</t>
 
+    <t> <list style="symbols">
+        <t>All information of priming response from Root server should be authenticated 
+          with DNSSEC. Not only the NS RR and DNSEKY, but also AAAA RR in additional 
+          section. Currently root-servers.net is not signed.</t>
+        <t>Root zone should be authoritative for root name so that AAAA RR will be 
+          included in prime response as authoritative results. Currently the IANA root 
+          zone is not authoritative for root name</t>
+        <t>To produce larger response packet to try the limit of the system regrading 
+          more root servers added. In IANA root practice the limit of 512B is still 
+          observed</t>
+      </list></t>
+
+      <t>Currently, there are two naming schemes as experiments in Yeti testbed. 
+        One is to use separate and normal domains for root servers (shown in figure 2). 
+        It intentionally produces larger packets for priming responses without 
+        compression. Note that currently, the Yeti root has a priming response which 
+        is almost the same size as the IANA root. Yeti has no compression, and has two 
```

Comment:
I just checked. Our priming response is 881 bytes, and the IANA root is 755 bytes. So we are larger.

Also, Yeti does have _some_ compression, since `dahu`.yeti.eu.org` and `dahu2.yeti.eu.org` have the same domain.